### PR TITLE
Fix lsc and vomses files creation, drop unused data.json legacy file

### PR DESCRIPTION
**Pull request by gwarf**
Various fixes and cleaning:
- Drop unused `data.json` legacy file
- Fix lsc and vomses file creation, to addapt to new yaml structure.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# https://github.blog/2017-07-06-introducing-code-owners/
+
+# Assign code owners that will automatically get asked to review Pull Requests
+# The last matching pattern takes the most precedence.
+
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for
+# review when someone opens a pull request.
+* @EGI-Federation/admins @EGI-Federation/operations
```
Comment (enolfc) [inline CODEOWNERS:10]:
```suggestion
* @EGI-Federation/admins @EGI-Federation/sdis
```

File: tasks/cache_config.yml [hunk 2]
```diff
@@ -25,16 +25,22 @@
...
-    dest: "{{ vomses_dir }}/{{ item.name }}-{{ item.voms.hostname }}"
+    dest: "{{ vomses_dir }}/{{ item.name }}-{{ item.hostname }}"
     content: >
-                        "{{ item.name }}" "{{ item.voms.hostname}}" "{{ item.voms.port }}"
-                        "{{ item.voms.DN }}" "{{ item.name }}"
-  loop: "{{ vo_voms | json_query('data[*]')}}"
+      "{{ item.name }}" "{{ item.hostname}}" "{{ item.port }}" "{{ item.DN }}"
+      "{{ item.name }}"
+  loop:
+    "{{ vo_voms | json_query('data[*]') | subelements('voms') | to_json |
```
Comment (enolfc) [inline tasks/cache_config.yml:44]: 🤯 let's assume you know what you are doing

**Review by enolfc — CHANGES_REQUESTED** [review]
just the codeowners to be fixed

File: tasks/cache_config.yml (hunk 2 above)
Comment (gwarf) [reply tasks/cache_config.yml:44]: Thanks stackoverfow ;)
This one was a real pain to debug, probably too complex, but at least I was able to get voms-proxy from the dteam VO that got configured with this ino...

**Review by enolfc — APPROVED** [review]

**Pull request merged**